runtime/moonbase/src/governance/tracks.rs

```diff
@@ -0,0 +1,286 @@
+			prepare_period: 4,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_REFERENDUM_CANCELLER,
+			min_support: SUP_REFERENDUM_CANCELLER,
+		},
+	),
+	(
+		13,
+		pallet_referenda::TrackInfo {
+			name: "referendum_killer",
+			max_deciding: 1_000,
+			decision_deposit: 50 * KILOUNIT * SUPPLY_FACTOR,
+			prepare_period: 4,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_REFERENDUM_KILLER,
+			min_support: SUP_REFERENDUM_KILLER,
+		},
+	),
+	(
+		14,
+		pallet_referenda::TrackInfo {
+			name: "small_tipper",
+			max_deciding: 200,
+			decision_deposit: 500 * UNIT * SUPPLY_FACTOR,
+			prepare_period: 4,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 28 * DAYS,
+			min_approval: APP_SMALL_TIPPER,
+			min_support: SUP_SMALL_TIPPER,
+		},
+	),
+	(
+		15,
+		pallet_referenda::TrackInfo {
+			name: "big_tipper",
+			max_deciding: 100,
+			decision_deposit: 5 * KILOUNIT * SUPPLY_FACTOR,
+			prepare_period: 4,
+			decision_period: 28 * DAYS,
+			confirm_period: 6 * HOURS,
+			min_enactment_period: 28 * DAYS,
+			min_approval: APP_BIG_TIPPER,
+			min_support: SUP_BIG_TIPPER,
+		},
+	),
+	(
+		16,
+		pallet_referenda::TrackInfo {
+			name: "small_spender",
+			max_deciding: 50,
+			decision_deposit: 50 * KILOUNIT * SUPPLY_FACTOR,
+			prepare_period: 4,
+			decision_period: 28 * DAYS,
+			confirm_period: 12 * HOURS,
+			min_enactment_period: 28 * DAYS,
+			min_approval: APP_SMALL_SPENDER,
+			min_support: SUP_SMALL_SPENDER,
+		},
+	),
+	(
+		17,
+		pallet_referenda::TrackInfo {
+			name: "medium_spender",
+			max_deciding: 20,
+			decision_deposit: 150 * KILOUNIT * SUPPLY_FACTOR,
+			prepare_period: 4,
+			decision_period: 28 * DAYS,
+			confirm_period: 24 * HOURS,
+			min_enactment_period: 28 * DAYS,
+			min_approval: APP_MEDIUM_SPENDER,
+			min_support: SUP_MEDIUM_SPENDER,
+		},
+	),
+	(
+		18,
+		pallet_referenda::TrackInfo {
+			name: "big_spender",
+			max_deciding: 10,
+			decision_deposit: 500 * KILOUNIT * SUPPLY_FACTOR,
+			prepare_period: 4,
+			decision_period: 28 * DAYS,
+			confirm_period: 48 * HOURS,
+			min_enactment_period: 28 * DAYS,
+			min_approval: APP_BIG_SPENDER,
+			min_support: SUP_BIG_SPENDER,
+		},
+	),
+];
+
+pub struct TracksInfo;
+impl pallet_referenda::TracksInfo<Balance, BlockNumber> for TracksInfo {
+	type Id = u16;
+	type Origin = <Origin as frame_support::traits::OriginTrait>::PalletsOrigin;
+	fn tracks() -> &'static [(Self::Id, pallet_referenda::TrackInfo<Balance, BlockNumber>)] {
+		&TRACKS_DATA[..]
+	}
+	fn track_for(id: &Self::Origin) -> Result<Self::Id, ()> {
+		if let Ok(system_origin) = frame_system::RawOrigin::try_from(id.clone()) {
+			match system_origin {
+				frame_system::RawOrigin::Root => Ok(0),
+				_ => Err(()),
+			}
+		} else if let Ok(custom_origin) = origins::Origin::try_from(id.clone()) {
+			match custom_origin {
```

Comment:
Should this have a default branch to induce an Err just to be safe?

Reply:
The rust compiler would complain if the match did not have a branch for all enum variants.

I don't even think we can add an additional branch without the compiler complaining (because it already has a branch for every variant of the origins::Origin enum